frontend_build/src/extract_$trs.js

```diff
@@ -67,18 +70,19 @@ extract$trs.prototype.apply = function(compiler) {
             }
           });
           if (messageNameSpace) {
-            // Every message needs to be namespaced - don't pollute our top level!
-            Object.keys(messages).forEach(function(key) {
-              // Create a new message id from the name space and the message id joined with '.'
-              var msgId = messageNameSpace + '.' + key;
-              if (messageExport[msgId]) {
-                // Warn about duplicate ids *within* a bundle (no way to warn across).
-                logging.error('Duplicate translation id ' + msgId + ' found in ' + module.resource)
```

Comment:
If we check for duplicate namespaces, would there be a need to check for duplicate msgIds since objects cannot have duplicate keys? I might be misunderstanding something.